sources/packages/forms/src/form-definitions/sfaa2022-23.json

```diff
@@ -2514,7 +2514,7 @@
                               "tableView": true,
                               "validate": {
                                 "required": true,
-                                "custom": "// To calculate the time difference of two dates\r\nconst Difference_In_Time =\r\n  new Date(data.studyendDate).getTime() -\r\n  new Date(data.studystartDate).getTime();\r\n\r\n// To calculate the no. of days between two dates\r\nconst Difference_In_Days = Difference_In_Time / (1000 * 3600 * 24);\r\nvalid = data.studystartDate\r\n  ? Difference_In_Days >= data.studyPeriodMinDays &&\r\n    Difference_In_Days <= data.studyPeriodMaxDays\r\n    ? true\r\n    : `Invalid Study Period, Dates must be between ${data.studyPeriodMinDays} and ${data.studyPeriodMaxDays} days`\r\n  : \"Please provide Study Start date\";",
+                                "custom": "// To calculate the time difference of two dates\r\nconst Difference_In_Time =\r\n  new Date(data.studyendDate).getTime() -\r\n  new Date(data.studystartDate).getTime();\r\n\r\n// To calculate the no. of days between two dates\r\nconst Difference_In_Days = Difference_In_Time / (1000 * 3600 * 24);\r\n// To calculate the no. of weeks between two dates\r\nconst Difference_In_Weeks = Difference_In_Days / 7;\r\nvalid = data.studystartDate\r\n  ? Difference_In_Time >= 0 \r\n    ? Difference_In_Weeks >= data.studyPeriodMinWeeks && Difference_In_Days <= data.studyPeriodMaxDays\r\n      ? true\r\n      : `Invalid Study Period, ${data.offeringIntensityText} programs must be at least ${data.studyPeriodMinWeeks} weeks and less than ${data.studyPeriodMaxDays} days`\r\n    : \"Study Start date must be before Study End date\"\r\n  : \"Please provide Study Start date\";",
```

Comment:
The requirement in the ticket mention the removal of existing minimum validation in PIR section of student application and not to replace it with weeks. 

![image](https://github.com/user-attachments/assets/7af2a17a-d6a7-4589-9adc-9ac568a79665)